device/alpaka/include/traccc/alpaka/seeding/track_params_estimation.hpp

```diff
@@ -0,0 +1,71 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s)
+#include "traccc/alpaka/utils/definitions.hpp"
```

Comment:
Do you actually need these definitions in the header file? Don't you only need them in the `.cpp` file?

This is what I meant earlier when pondering whether it would be possible to move `definitions.hpp` under `src/`. Since I don't think you need its definitions in the public interface of any of the classes / algorithms.